test/cli.js

```diff
@@ -0,0 +1,165 @@
+import { Launcher } from '@wdio/cli';
```

Comment:
This one can be a bit hard to understand without digging into https://www.npmjs.com/package/yargs. Type `./test.sh` when on this branch to see help for the CLI options this provides.